_pages/install/installing-sap-commerce-cloud.md

```diff
@@ -11,7 +11,9 @@ The steps described in this document are:
 1. Set up SAP Commerce Cloud
 2. Configure OCC credentials
 3. Configure CORS (optional to start but required for checkout)
-4. Install the Spartacus sample data addon (optional)
+4. Install the Spartacus sample data AddOn (optional)
+
+    **Note:** If you are working with the default implementation of Spartacus, it is recommended that you install the Spartacus sample data AddOn. By default, there is a tight relation between the `B2cStorefrontModule` and the Spartacus sample data. However, when you start your own project, you will have your own sample data, and at that point, you should use the `StorefrontModule` (rather than the `B2cStorefrontModule`) when you create your own pages, templates and components. If you are using your own sample data, you do not need to install the Spartacus sample data.
```

Comment:
I would express this slightly different, but I leave it to you to mix the 2 descriptinos if you feel the result is better: 

> While the Spartacus layout is CMS driven, there are a few areas where the CMS structure doesn't provide enough information. Therefor, there is a Layout configuration that provides additional information for the layout rendering of CMS content (specifically the order of page slots). This configuration is provided in the `B2cStorefrontModule`. It is important to understand that this specific configuration is tightly coupled to the Spartacus sample data; whenever you change the sample data (something that happens in all projects), you should introduce your own layout configuration. In this case, you would not import the `B2cStorefrontModule`, but rather use the `StorefrontModule` that does not provide any layout configuration. The `StorefrontModule` is not depending on the Spartacus sample data, and is most likely a good start for your project.

Reply:
This is excellent! Thank you very much!
I took your suggested content and just made a few minor tweaks 🙂 
Let me know if this is good...